README.md

```diff
@@ -53,30 +53,51 @@ Will validate TypeScript code in the project. This requires a `tsconfig.json` fi
 
 This package contains a configuration of [`semantic-release`](semantic-release.gitbook.io/), which can be used for automated software releases, published on Github. It's configured to work with the following repository branch setup:
 
-1. `master` – ongoing development
+1. `trunk` – ongoing development
 1. `alpha` – release candidate
+1. `hotfix/*` - for testing urgent bugfixes
+1. `epic/*` - for testing large-scale features
 1. `release` – the production-ready, released code
 
 The following assumes that CI will run:
 
-1. `npm run release` for `release`, `alpha`, and `hotfix/*` branches
+1. `npm run release` for `release`, `alpha`, `hotfix/*`, and `epic/*` branches
 1. `post-release.sh` script on `release` branch, after the above command completes
 
 ### Regular release flow
 
-1. Commit ongoing changes to `master` branch, using [structured commit messages](https://www.conventionalcommits.org/en/v1.0.0/)
-1. Merge `master` into `alpha` to create a release candidate (e.g. `1.2.0-alpha.1`)
-1. Merge `alpha` into `release` to create a release (e.g. `1.2.0`)
-1. `alpha` branch will be reset on top of `release`
-1. `master` branch will be updated with the changes from the `release` branch
+1. Create a new `feat/*` or `fix/*` (or `chore/*` or `style/*`) branch off the `trunk` branch.
```

Comment:
I think that's too prescriptive, and do we even follow this guideline? I rarely even look at the branch names of "feature branches".

Reply:
Ok I do follow this almost unconsciously :D  But I still think it's too prescriptive.

Reply:
Sounds good! You're right that the names of the feature/fix branches actually don't matter in our workflow. Only `trunk`/`alpha`/`hotfix/*`/`epic/*` branch names are significant. Updated in 9d20b2d.